taf/updater/updater_pipeline.py

```diff
@@ -401,29 +406,26 @@ def clone_remote_and_run_tuf_updater(self):
                     )
                 else:
                     self.state.auth_repo_name = auth_repo_name
-
             self.state.users_auth_repo = AuthenticationRepository(
-                self.library_dir,
-                self.state.auth_repo_name,
+                path=Path(self.library_dir) / self.state.auth_repo_name,
```

Comment:
You should not have to do that, `path` is set automatically when you specify `library` and `name`, There's an issue somewhere if that is not the case atm